models/engine/db_storage.py

```diff
@@ -90,3 +90,22 @@ def close(self):
             calls remove() on private session attribute (self.session)
         """
         self.__session.remove()
+
+    def get(self, cls, id):
+        """
+            retrieves one object based on class name and id
```

Comment:
I like this comment style here.  Would you want to make this uniform for all files?  How about we can start with the Storage Engine Files.